docs/program-setup/serial/serial_communication.md

```diff
@@ -6,86 +6,100 @@ The Arm Mbed microcontroller on your board can communicate with a host PC over t
 
 This allows you to:
 
-- Print out messages to a [host PC terminal (useful for debugging)](#terminal-applications).
-- Read input from the host PC keyboard.
-- Communicate with applications and programming languages running on the host PC that can communicate with a serial port. Examples are Perl, Python and Java.
+- Print out messages to a PC terminal (useful for debugging).
+- Read input from a PC keyboard.
+- Communicate with applications running on a PC to exchange data.
 
-## Hello, world
+## Hello World - printing messages
 
-This program prints a "Hello World" message that you can view on a [terminal application](#using-terminal-applications). Communication over the USB serial port uses the standard serial interface. Specify the internal (USBTX, USBRX) pins to connect to the serial port routed over USB:
+This program prints a "Hello World!" message that you can view on a serial terminal. Mbed OS redirects any `printf()` statements to the board's debug USB serial.
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-snippet-Serial_HelloWorld/tree/v6.0)](https://github.com/ARMmbed/mbed-os-snippet-Serial_HelloWorld/blob/v6.0/main.cpp)
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-snippet-Serial_STDOUT/tree/v6.0)](https://github.com/ARMmbed/mbed-os-snippet-Serial_STDOUT/blob/v6.0/main.cpp)
+
+## Viewing output in a serial terminal
+
+Serial terminals run on your host PC. They provide an interface where your Mbed board can print and where you can type characters back to your board.
+
+Mbed CLI provides a serial terminal that is configured with a default baud rate of `9600`. When a single board is connected, run:
 
-## Using terminal applications
+```
+mbed sterm
```

Comment:
The most common usage is auto-opening a serial terminal after compilation so I would put that section first and describe the sterm command subsequently.

Reply:
Good point, I just reordered and rephrased to introduce auto-opening after compilation first.